activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -354,17 +354,24 @@ def group!(*args) # :nodoc:
       self
     end
 
-    # Allows to specify an order attribute:
+    # Order can be used to sort the results of a query.
     #
+    # #order accepts arguments in one of several formats.
+    #
+    # === symbols
```

Comment:
I think we always add an extra new line after a heading:
```suggestion
    # === symbols
    #
```
Same thing applies to the other two headings.